tests/fixtures/phpdoc_tags_inline.php

```diff
@@ -0,0 +1,78 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * A fixture to verify various phpdoc tags are allowed inline.
+ *
+ * @package   local_moodlecheck
+ * @copyright 2020 onwards Eloy Lafuente (stronk7) {@link http://stronk7.com}
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
```

Comment:
I always add that line into fixtures because they can have executable code that we don't want to execute ever. In this case the file only has that one class and that {{global $CFG;}} so we may remove both (If we keep the global, then for sure the MOODLE_INTERNAL is needed, it's global state change).

So, in this case, and to avoid forgetting it when copying it to another fixture... I feel slightly inclined to keep it.

Reply:
(as you want in any case, I'm happy both keeping them or removing them, it's really developer-only code so hardly will be in any prod server ever - I hope)

Reply:
Then it's ok for me to keep it there! :-)